lighty-modules/lighty-bgp/src/test/java/io/lighty/modules/bgp/deployer/ModuleInitTest.java

```diff
@@ -103,12 +103,7 @@ static void shutdown() {
 
     @SuppressWarnings({"checkstyle:illegalCatch"})
```

Comment:
Remove SuppressWarnings annotation form here, and also from other shutdown/close methods. This annotation is not required now because you have removed `} catch (Exception e) {`

Reply:
annotation removed